### ISSUE TITLE
Can't un/favorite a tab with the keyboard.

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#83]
Originally reported on Google Code with ID 81

```
What steps will reproduce the problem?
1. Try to un/favorite a tab with the keyboard.

What is the expected output? What do you see instead?
Can't. Should be able to.

Please provide any additional information below.
A lot of the keyboard-only stuff is put into the context menu... figuring out how to
do this correctly could be somewhat tricky.
```

Reported by `pcxunlimited` on 2010-10-17 17:15:31

**Comment by Pauan** [comment]
```
Punting to 3.3...
```

Reported by `pcxunlimited` on 2010-10-26 10:48:15
- **Labels added**: Milestone-3.3
- **Labels removed**: Milestone-3.2

**Comment by Pauan** [comment]
```
I really want to make this keyboard-accessible, but it's a little tricky figuring out
the best way. My current idea is to add "Favorite" and "Unfavorite" items to the "Selected..."
submenu.
```

Reported by `pcxunlimited` on 2010-11-08 08:41:50
- **Labels removed**: Milestone-3.3

**Comment by Pauan** [comment]
```
This issue was updated by revision e71ad618bd.
```

Reported by `pcxunlimited` on 2010-12-05 11:57:25
- **Status changed**: `Pending`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-12-13 01:00:59
- **Status changed**: `Verified`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:44:48
- **Status changed**: `Fixed`
